common/wonders/00_wonders.txt

```diff
@@ -2067,15 +2067,21 @@ wonder_garden = {
 wonder_university = {
 	picture = GFX_wonder_strip_university
 	potential = {
-		# Warcraft
-		evil_public_religion_trigger = no
-		
-		# Warcraft
-		is_tribal = no
-		
 		is_nomadic = no
-
-		# Warcraft
+		OR = {
+			religion_group = christian
```

Comment:
So, you can't now build university/library?

Reply:
Fixed!